openstack-ansible

```diff
@@ -0,0 +1 @@
+Subproject commit 4f779033bc199bd2dadcd554ac7d61c80cd39617
```

Comment:
@nrb @odyssey4me shouldn't this be the same sha upstream? The goal is RC18 is the same as RC17, where the only addition testing is to ensure the rename is fine.